test/integration/test_timeout.py

```diff
@@ -0,0 +1,224 @@
+#  © Copyright IBM Corporation 2020.
```

Comment:
nit wrong year, I also think we should add the `coding: utf-8` because of the © symbol
```suggestion
# coding: utf-8

# © Copyright IBM Corporation 2021.
```

Reply:
Added in 24bad01